extension/src/popup/views/AddAccount/AddAccount.tsx

```diff
@@ -46,17 +50,27 @@ export const AddAccount = () => {
     }
   };
 
+  const clearAddAccountError = (e: React.ChangeEvent<any>) => {
+    if (authError && e.target.value === "") {
+      dispatch(clearApiError());
+    }
+  };
```

Comment:
this code is a bit duplicative, but I wasn't able to figure out a way to reuse a hook on callback quickly. I've created a ticket for further investigation